src/components/BorrowerProfile/LendCta.vue

```diff
@@ -129,7 +129,7 @@
 											:amount-left="unreservedAmount"
 											@add-to-basket="addToBasket"
 											:complete-loan="isCompleteLoanActive"
-											v-if="isLendAmountButton"
+											v-if="isLendAmountButton && !enableFiveDollarsNotes"
```

Comment:
Is this button never needed for $5 notes?

Reply:
yeah, the needed one is above.